hashtagmap-admin/front/src/store/modules/zone.js

```diff
@@ -0,0 +1,184 @@
+      state.selectedDistrictName = selected;
+    },
+    CLEAR_SELECT_DISTRICT_NAME: state => {
+      state.selectedDistrictName = "";
+    }
+  },
+  actions: {
+    setZone: async ({ commit, dispatch, state }) => {
+      let snackbarContents = SNACK_BAR_TEMPLATE.SUCCESS();
+
+      try {
+        state.zoneInput.districtName = state.selectedDistrictName;
+        const newZone = state.zoneInput;
+        if (!newZone) {
+          snackbarContents = SNACK_BAR_TEMPLATE.INFO(MESSAGE.NO_INPUT);
+          return snackbarContents;
+        }
+        await customAxios().post("/districts/zones", newZone, {
+          headers: {
+            "Content-Type": "application/json"
+          }
+        });
+        commit("CLEAR_ZONE_INPUT");
+        dispatch("fetchZones");
+        return snackbarContents;
+      } catch (error) {
+        snackbarContents = SNACK_BAR_TEMPLATE.ERROR(error);
+      }
+      return snackbarContents;
+    },
+    removeZones: async ({ dispatch }, selectedZones) => {
+      if (!selectedZones || selectedZones.length === 0) {
+        return;
+      }
+      const zoneIds = {
+        zoneIds: selectedZones.map(zone => zone.zoneId)
+      };
+      await customAxios().delete("/districts/zones", {
+        headers: {
+          "Content-Type": "application/json"
+        },
+        data: zoneIds
+      });
+
+      dispatch("fetchZones");
+    },
+    modifyZone: async ({ state }) => {
+      const target = state.editTargetZone;
+      delete target.districtId;
+      target.districtName = state.selectedDistrictName;
+      console.log(target);
+      let snackbarContents = SNACK_BAR_TEMPLATE.SUCCESS();
+      try {
+        await customAxios().patch("/districts/zones", target, {
+          headers: {
+            "Content-Type": "application/json"
+          }
+        });
+      } catch (error) {
+        snackbarContents = SNACK_BAR_TEMPLATE.ERROR(error);
+      }
+      return snackbarContents;
+    },
+    fetchZones: async ({ commit }) => {
+      let snackbarContents = SNACK_BAR_TEMPLATE.INFO(MESSAGE.NO_CONTENT);
+      try {
+        const response = await customAxios().get("/districts/zones");
+        commit("CLEAR_ZONES");
+        const responseZones = response.data.data;
+        if (responseZones.length === 0) {
+          return snackbarContents;
+        }
+        responseZones.map(zone => commit("ADD_ZONE", zone));
+        snackbarContents = SNACK_BAR_TEMPLATE.SUCCESS();
+      } catch (error) {
+        snackbarContents = SNACK_BAR_TEMPLATE.ERROR(error);
+      }
+      return snackbarContents;
+    },
+    fetchDistrictNames: async ({ commit }) => {
+      const response = await customAxios().get("/districts/names");
+      commit("CLEAR_DISTRICT_NAME");
+      const responseNames = response.data.data;
+      if (responseNames.length === 0) {
+        return;
+      }
+      responseNames.map(name => commit("ADD_DISTRICT_NAME", name));
+    }
+  }
```

Comment:
조금 헷갈렸던 부분이 있는데 비밥은 네이밍 컨벤션에서 fetch와 set의 차이에 대해 어떻게 생각하는지 궁금합니다

```
Good examples:

fetchProduct - Gets product by ID from server or cache, sets it in products map, and returns it by getter.
findProducts - Fetches products by specific query, sets them in products map, and returns them as array.
setCurrentProduct - Param could be ID, it could dispatch fetchProduct, mutate it to productsMap, and mutate its ID to currentProductId. Also if productId is null, then it removes currentProduct.
addCartItem
toggleMicrocart
```

일단 나는 그냥 정보를 요청해서 get만하는거는 fetch를 쓰고 가져온 값으로 state에 적용하는거는 set을 해야하나 생각은 했는데 한번 얘기나눠봐요